tools/crates-validator/src/profiles.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2023 RISC Zero, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::CrateProfile;
+
+pub const SKIP_CRATES: &[&str] = &[
+    "miow", // Windows specific crate that depends on `windows-sys`
+    "windows-sys",
+    "redox_syscall", // Tools for redox OS, not supported
```

Comment:
Not blocking, but if it's easy to have these comments appear when you click "Skipped" in the same way the error logs appear when you click "BuildFail", that would be a nice addition.

Reply:
Yeah probably a good call, I can probably make this a tuple of (crate_name, "description str") and then return that into the results if_skipped.